pkg/services/librarypanels/api.go

```diff
@@ -72,3 +73,20 @@ func (lps *LibraryPanelService) getAllHandler(c *models.ReqContext) api.Response
 
 	return api.JSON(200, util.DynMap{"result": libraryPanels})
 }
+
+// updateHandler handles PUT /api/library-panels/:uid
+func (lps *LibraryPanelService) updateHandler(c *models.ReqContext, cmd updateLibraryPanelCommand) api.Response {
+	cmd.UID = c.Params(":uid")
```

Comment:
We should be consistent with the other methods, and pass UID as a separate parameter. It's also less confusing to readers of the code if `cmd` corresponds 1:1 to request parameters (single responsibility).

Reply:
Yes, I totally agree.

Reply:
Fixed